packages/chrome-extension/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@clerk/chrome-extension",
+  "version": "0.1.0",
+  "types": "./dist/types/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/esm/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsup --env.NODE_ENV production",
+    "dev": "tsup --watch",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@clerk/clerk-js": "^4.34.0-staging.0",
+    "@clerk/clerk-react": "^4.14.0-staging.0"
+  },
+  "devDependencies": {
+    "@testing-library/dom": "^8.19.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/chrome": "*",
+    "@types/node": "^16.11.55",
+    "@types/react": "*",
+    "@types/react-dom": "*",
+    "jest": "^29.3.0",
+    "jest-environment-jsdom": "^29.3.1",
+    "ts-jest": "^29.0.3",
+    "tsup": "*",
+    "typescript": "*"
+  },
+  "peerDependencies": {
+    "react": ">=16"
+  },
+  "eslintConfig": {
```

Comment:
Do we need this? We already have the top-level eslint config set up. Is it not working for you?

Reply:
nice catch. i will remove it.